### PR TITLE
alive-progress 3.1.5 ❄️ 

### PR DESCRIPTION
**Pull request by psteyer**
alive-progress 3.1.5 ❄️ 

**Destination channel:** Snowflake

### Links

- [PKG-4233](https://anaconda.atlassian.net/browse/PKG-4233) 
- [Upstream repository](https://github.com/rsalmei/alive-progress)

### Explanation of changes:

- adding skip statement for python
- adding host requirements
- added extra about
- adding script flags

[PKG-4233]: https://anaconda.atlassian.net/browse/PKG-4233?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,19 +8,21 @@ package:
...
     - pip
-    - python >=3.7,<4
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<4
+    - python
     - about-time ==4.2.1
     - grapheme ==0.6.0
```
Comment (lorepirri) [inline recipe/meta.yaml:27]: You can eventually relax the pinnings here:

```suggestion
    - about-time >=4.2.1
    - grapheme >=0.6.0
```

**Review by lorepirri — APPROVED** [review]
LGTM! Eventually, relax the pinnings

File: recipe/meta.yaml (hunk 1 above)
Comment (JamesRobertsonGames) [reply recipe/meta.yaml:27]: https://github.com/rsalmei/alive-progress/blob/main/setup.py#L60

The versions are pinned here, we should keep it pinned for the time being unless they change this

**Review by JamesRobertsonGames — APPROVED** [review]
Looks good to me! :+1:

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml:27]: @JamesRobertsonGames @psteyer https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Pinnings.md#dont-necessarily-follow-what-upstream-does-to-the-letter

**Pull request merged**